docs/intro/quickstart/access.rst

```diff
@@ -0,0 +1,423 @@
+.. _ref_quickstart_access:
+
+=====================
+Adding Access Control
+=====================
+
+.. edb:split-section::
+
+  Let's add a concept of a user to our application, and update our data model to limit access to the decks and cards to only the user's own decks. Our ``User`` type will be very simple, and for authentication we will use a simple ``AccessToken`` type that gets returned from the user creation endpoint when you make a new user. Gel has some really powerful tools available in our authentication extension, but for now we will just use a simple token that we will store in the database.
+
+  Along with this user type, we will add some ``global`` values that will use the access token provided by the client to set a global ``current_user`` variable that we can use in our queries to limit access to the decks and cards to only the user's own decks.
+
+  .. note::
+
+    Deck creators should be required, but since we are adding this to an existing dataset, we will set the new ``creator`` property to optional. That will effectively make the existing cards and decks invisible since they don't have a creator. We can update the existing data in the database to set the ``creator`` property for all of the existing decks and cards after making the first user, or reinsert the deck and the creator will be set in our updated query.
+
+  .. code-block:: sdl-diff
+    :caption: dbschema/default.gel
+
+      module default {
+    +   single optional global access_token: uuid;
+    +   single optional global current_user := (
+    +     select AccessToken filter .id = global access_token
+    +   ).user;
+    +
+    +   type User {
+    +     required name: str;
+    +   }
+    +
+    +   type AccessToken {
+    +     required user: User;
+    +   }
+    +
+        type Deck {
```

Comment:
We have to split this long example into smaller blocks -- one talking about adding just `single optional global access_token` line, adding a sentence on what it does, etc.